### PR TITLE
chore: Apply linting and ignore new bazel dirs

### PR DESCRIPTION
**Pull request by narendasan**
Signed-off-by: Naren Dasan <naren@narendasan.com>
Signed-off-by: Naren Dasan <narens@nvidia.com>

# Description

Fixing the linter speed

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [x] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes
- [x] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: core/conversion/conversionctx/ConversionCtx.cpp [hunk 1]
```diff
@@ -124,13 +124,13 @@ ConversionCtx::ConversionCtx(BuilderSettings build_settings)
         settings.enabled_precisions.find(nvinfer1::DataType::kFLOAT) == settings.enabled_precisions.end(),
         "DLA supports only fp16 or int8 precision");
     cfg->setDLACore(settings.device.dla_core);
-    if (settings.dla_sram_size != 1048576){
+    if (settings.dla_sram_size != 1048576) {
```
Comment (andi4191) [inline core/conversion/conversionctx/ConversionCtx.cpp:127]: @narendasan : I see hard-coded values at multiple places. Can we move this to some macro definition instead?

**Review by andi4191 — APPROVED** [review]

File: core/conversion/conversionctx/ConversionCtx.cpp (hunk 1 above)
Comment (narendasan) [reply core/conversion/conversionctx/ConversionCtx.cpp:127]: Sure thats fine, can you do this in a separate PR?

**Review by peri044 — APPROVED** [review]
Now the linter runs fine for me.

**Comment by peri044** [comment]
Python linter also runs fine (without hanging). However, python linter changes haven't been added to this PR.

File: core/conversion/conversionctx/ConversionCtx.cpp (hunk 1 above)
Comment (andi4191) [reply core/conversion/conversionctx/ConversionCtx.cpp:127]: Yeah, I will do that.

**Comment by narendasan** [comment]
@peri044 @andi4191 Can you try this PR out? Likely this will be the last PR to merge before release since it is so disruptive but I want to verify that you guys are able to set up and use the new system. Try making a trivial documentation change and push to this PR to test it out.

File: third_party/tensorrt/local/BUILD [hunk 2]
```diff
@@ -351,11 +351,6 @@ cc_library(
         ":ci_rhel_x86_64_linux": glob(["lib64/libnvinfer_plugin.so"]),
         "//conditions:default": ["lib/x86_64-linux-gnu/libnvinfer_plugin.so"],
     }),
-    hdrs = select({
```
Comment (narendasan) [inline third_party/tensorrt/local/BUILD]: Why did this get removed?

Comment (andi4191) [reply third_party/tensorrt/local/BUILD]: There was a conflict with the base branch. I resolved it by replacing it with the ci_rhel_x86_64_linux version

**Pull request merged**